common/utils/src/main/java/org/graalvm/buildtools/agent/AgentConfiguration.java

```diff
@@ -57,6 +57,18 @@ public class AgentConfiguration implements Serializable {
 
     private final AgentMode agentMode;
 
+    // This constructor should be used only to specify that we have instance of agent that is disabled (to avoid using null for agent enable check)
+    public AgentConfiguration() {
+        this.callerFilterFiles = null;
+        this.accessFilterFiles = null;
+        this.builtinCallerFilter = false;
+        this.builtinHeuristicFilter = false;
+        this.experimentalPredefinedClasses = false;
+        this.experimentalUnsafeAllocationTracing = false;
+        this.trackReflectionMetadata = false;
```

Comment:
The fields already have those values by default. Remove.

Reply:
This class is also used by gradle plugin, so I am not sure if we can remove default values.

Reply:
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html